fhir-core/src/main/java/org/linuxforhealth/fhir/core/context/FHIRPagingContext.java

```diff
@@ -64,13 +64,13 @@ public interface FHIRPagingContext {
      * 
      * @return the resource Id of the first resource of the next search result
```

Comment:
we should probably update the javadoc to indicate what we mean by "resource Id" here

Reply:
```suggestion
     * @return the internal resource id of the first entry of the current page; this id should
     *        uniquely identify a particular version of a particular resource type with a particular logical id
```